gensrc/proto/olap_file.proto

```diff
@@ -88,6 +88,7 @@ enum PartialUpdateMode {
     UNKNOWN_MODE = 0;
```

Comment:
UNKNOWN is enough for this name.

Reply:
`UNKNOWN` will be duplicate name with other enum in namespace `starrocks` , and cause compile error. So I think it is OK to use this.